addon/adapters/github-repository-contents.js

```diff
@@ -0,0 +1,12 @@
+import GitHubAdapter from './github';
```

Comment:
I prefer to explicit on these imports so that any file moves don't surprise us. Could you update this and all other new imports to have `ember-data-github/<thing>/<name>` in them